src/test/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/ProfileTest.java

```diff
@@ -0,0 +1,42 @@
+package io.github.oliviercailloux.y2018.apartments.valuefunction.profile;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.google.common.collect.BoundType;
+import com.google.common.collect.Range;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.github.oliviercailloux.y2018.apartments.valuefunction.Criterion;
+
+public class ProfileTest {
+
+	Profile profile;
+	
+	 /** Inits the profile object before each tests */
+	  @BeforeEach
+	  void initEach() {
+		  Profile.Builder profileBuilder = new Profile.Builder();
+		  
+		  for (Criterion c : Criterion.values()) {
+		      profileBuilder.setWeightRange(c, Range.range(Double.valueOf(10d), BoundType.CLOSED, Double.valueOf(0d), BoundType.CLOSED));
```

Comment:
Problème dans l'ordre des bornes du range. Par ailleurs, est-il vraiment nécessaire d'utiliser `Double.valueOf()`

Reply:
Il est nécessaire d'utiliser Double.valueOf, il s'agit d'une factory pour les objets Double. Nous utilisons des objet de type Double et non double car nous utilisons des Range<Double> et la classe Range a besoin de types immuable pour fonctionner.

Reply:
Fixed tests issues in commit cd92288